eox_nelp/signals/tests/test_tasks.py

```diff
@@ -42,12 +45,17 @@ def test_call_user_has_passing_grade(self, course_grade_factory_mock):
         course_grade_factory_mock().read.assert_called_with(user, course_key=CourseKey.from_string(course_id))
 
 
+@ddt
 class DipatchFuturexProgressTestCase(unittest.TestCase):
     """Test class for function `dispatch_futurex_progress`"""
 
+    @patch("eox_nelp.signals.tasks.settings")
```

Comment:
please don't mock settings, this will affect whole default behavior since now everything is a mock , you can use` @override_settings` decorator or use `setattr` to change a specific setting value

Reply:
[b58dfd3](https://github.com/eduNEXT/eox-nelp/pull/41/commits/b58dfd328232bba40e31b59657dadd042cb8039e)